synthnn/core/musical_extensions.py

```
@@ -7,12 +7,13 @@
 """
 
 import numpy as np
-from typing import Dict, List, Tuple, Optional, Any
+from typing import Dict, List, Tuple, Optional, Any, Union
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)